src/ResourceManager/MachineLearning/AzureRM.MachineLearning.psd1

```diff
@@ -0,0 +1,110 @@
+#  
+# Module manifest for module 'Microsoft.Azure.Commands.MachineLearning'  
+#  
+# Generated by: Microsoft Corporation  
+#  
+# Generated on: 3/7/2016
+#  
+  
+@{  
+  
+# Version number of this module.  
+ModuleVersion = '0.9.0' 
+  
+# ID used to uniquely identify this module  
+GUID = 'BB030259-49F9-46EF-806F-2A3F5A2E018E'  
+  
+# Author of this module  
+Author = 'Microsoft Corporation'  
+  
+# Company or vendor of this module  
+CompanyName = 'Microsoft Corporation'  
+  
+# Copyright statement for this module  
+Copyright = 'Microsoft Corporation. All rights reserved.'    
+  
+# Description of the functionality provided by this module  
+Description = 'Microsoft Azure PowerShell - Machine Learning Web Services cmdlets for Azure Resource Manager'  
+  
+# Minimum version of the Windows PowerShell engine required by this module  
+PowerShellVersion = '3.0'  
+  
+# Name of the Windows PowerShell host required by this module  
+PowerShellHostName = ''  
+  
+# Minimum version of the Windows PowerShell host required by this module  
+PowerShellHostVersion = ''  
+  
+# Minimum version of the .NET Framework required by this module  
+DotNetFrameworkVersion = '4.0'  
+  
+# Minimum version of the common language runtime (CLR) required by this module  
+CLRVersion='4.0'  
+  
+# Processor architecture (None, X86, Amd64, IA64) required by this module  
+ProcessorArchitecture = 'None'  
+  
+# Modules that must be imported into the global environment prior to importing this module  
+RequiredModules = @( @{ ModuleName = 'AzureRM.Profile'; ModuleVersion = '1.0.8'})
```

Comment:
change this to 1.0.9. If this is the only change - no need to run another on-demand.